api/build/clean.go

```diff
@@ -19,6 +19,16 @@ import (
 // without execution. This will kill all resources,
 // like steps and services, for the build.
 func CleanBuild(ctx context.Context, database database.Interface, b *types.Build, services []*library.Service, steps []*library.Step, e error) {
+	logger := logrus.WithFields(logrus.Fields{
```

Comment:
no gin/middleware for this one? 
```
	l := c.MustGet("logger").(*logrus.Entry)
```

Reply:
should we call it `l` at least to be consistent

Reply:
i might have to throw in a check to see if ctx is gin or not, then we could. i just checked one path and it was, but the signature made me think it was not always.

Reply:
yea, this is not always gin.Context, so it won't have the logger in it in certain scenarios. opted to just rename to `l` for now. lmk, if you think it's worth doing a type check on context and pulling the logger out if we have it. https://github.com/go-vela/server/pull/1145/commits/924390dfb13b70146a4ddcb0fda4abc9bbe88f16